Makefile

```
@@ -207,7 +208,14 @@ setup-new-sdk:
 .PHONY: build-client-java
 build-client-java:
 	make build-client sdk_language=java tmpdir=${TMP_DIR}
-	make run-in-docker sdk_language=java image=gradle:${GRADLE_DOCKER_TAG} command="/bin/sh -c 'gradle fmt'"
+	make run-in-docker sdk_language=java image=gradle:${GRADLE_DOCKER_TAG} command="/bin/sh -c 'chmod +x ./gradlew && gradle fmt build'"
+
+.PHONY: test-client-java
```

Why do we have two .PHONY with test-client-java (line 213 and line 220)?

Also, will be good to update test-all-clients and build-all-clients with java so that the CICD will also exercise this as well.

---

Oops this .PHONY is incorrect

```suggestion
.PHONY: test-integration-client-java
```